models/order.js

```diff
@@ -12,32 +12,41 @@ const orderSchema = new Schema({
 });
 
 orderSchema.statics.getOrders = function() {
-  return getOrders.apply(this)
-  .then(getEntries)
-  .catch((e) => {
-    throw new Error(e);
```

Comment:
Whoops.. I'll add back. I should be able to catch in `transactAndDispatch` right because errors get passed on? Or do you think I should catch at each step?